Pipfile

```diff
@@ -8,6 +8,7 @@ django = "*"
 
 [dev-packages]
 flake8 = "*"
+pytest = "*"
```

Comment:
I think It should be pytest-django plugin:
https://pytest-django.readthedocs.io/en/latest/

Reply:
It should be both, see #34

Reply:
How did you decide that you need both?
because I see in Barak's example that only the pytest-django is needed. 
https://github.com/redhat-beyond/beyond-tutorial/blob/main/Pipfile

Reply:
in all example i saw online and past project there were both, even in the lesson on pytest if Im not mistaken

Reply:
@ronyyosef @adiabramovitch was this resolved?